controllers/reviews.controller.js

```diff
@@ -33,3 +33,14 @@ exports.patchReviewById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getReviewComments = (req, res, next) => {
```

Comment:
Before you invoke getReviewComments, I'd suggest invoking another function (you've already written) that will check whether an article exists, and throw an error if it doesn't